docs/specs/oidc.md

```diff
@@ -72,6 +72,7 @@ Supported [standard client metadata](https://openid.net/specs/openid-connect-reg
 - `client_id`: OIDC client ID.
 - `access_token_lifetime`: Access token lifetime in seconds, default to 1800.
 - `refresh_token_lifetime`: Refresh token lifetime in seconds, default to max(access_token_lifetime, 86400). It must be greater than or equal to `access_token_lifetime`.
+- `is_first_party`: Indicate whether the client is a [first-party client](#first-party-clients), default to false.
```

Comment:
Is it related with the "first party client" vs "first party application" wordings? Or update the naming later?

Reply:
I've opened #823 to track updates on usage of terms, since most likely we want to review/update the whole spec in one go.